doc: Rename Ecosystems integration section and template (#10705)
nrfconnect/sdk-nrf: doc/nrf/integrations.rst

.. _integrations:

Integrations
############

> An |NCS| Integration allows for using a third-party product from within the |NCS|.

Reviewer: If we say third party, would nRF Cloud fit here?

> An |NCS| Integration allows for using a third-party product from within the |NCS|.

Author: ```suggestion
An |NCS| Integration allows for using third-party and Nordic products within the |NCS|.
```
Since, nRF cloud is a Nordic product, may be we could change it as "third-party and Nordic products"? Thoughts?
If so, I can make the changes in the Integration template also.

Reviewer 2: I am ok with that.

> The following user guides describe available integrations.

Reviewer 3: Can we move line 7 after line 14 to align it better with the list of user guides?

Author: updated.

> The following user guides describe the integration of various third-party products with |NCS|.

Reviewer 2: ```suggestion
The following user guides describes available integrations.
```

> The |NCS| is also integrated with the following third-party products that are documented in private repositories:

Reviewer 2: ```suggestion
Integration of the following third-party products is documented in their private repositories:
```
